doc/ug/install_instructions/index.md

```diff
@@ -310,9 +310,9 @@ But unfortunately it doesn't support the batch mode for unattended installations
 
 To get started faster we use the web installer in the following.
 
-1. Go to the [Xilinx download page](https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/vivado-design-tools/2020-1.html) and download two files for Vivado 2020.1.
```

Comment:
The install instructions change slightly every version. Instead of trying to parametrize it and then making it slightly wrong, I'd just keep it at a single version that was actually tested (we also have screenshots in the instructions!) and then let people adjust as necessary.

Reply:
Yes, I think that makes most sense. Will change it accordingly.